src/Components/SearchOptions.jsx

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Button, Col, Form, Row } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import { getArticleTopics } from "../Requests/makeRequests";
+
+function SearchOptions({ handleClose }) {
+	const [topics, setTopics] = useState([]);
+	const [chosenTopic, setChosenTopic] = useState();
+
+	const navigate = useNavigate();
+
+	useEffect(() => {
+		getArticleTopics().then(({ data: { topics } }) => {
+			setTopics(topics);
+		});
+	}, []);
+
+	const handleSubmission = (e) => {
+		e.preventDefault();
+		navigate(`/search/${chosenTopic}`);
```

Comment:
Nice job getting a drop down menu to work to navigate to the correct page, it can be a bit tricky 👍